content/community/organization/policies.md

```diff
@@ -0,0 +1,110 @@
+|Term|Meaning|
+|---|---|
+|Community Member|The term "community member" is used in this document to refer to any individual who is engaged with the Haiku project in any way.|
+|Communications Channel|The term "communications channel" is intended to cover all communications channels related to the Haiku project.  Examples of "communications channels" include IRC, mailing lists, emails, code-reviews, discussion boards and other media.|
+|Development Team|The term "development team" is used to refer to a group of individuals who have commit access into the Haiku source code repository and who are recently actively engaged on the project contributing to the source code.|
+
+## Volunteers
+
+The Haiku project is contributed to by a number of people who volunteer their time.  These volunteer contributors often have busy lives and have to consider other activities and obligations in their lives ahead of following their interests in the Haiku project.  Often volunteer contributors have periods in their lives when it is feasible to engage with the Haiku project and times when it is not feasible.  Some volunteer contributors may contribute for a time and then stop contributing as their life priorities and interests change.
+
+Whilst a volunteer contributor may try to help and may be sympathetic to any given community member's questions or requests, a volunteer contributor has no actual obligation to the community or any community member to;
+
+- continue to provide volunteer contributions
+- provide contributions at any fixed frequency
+- to meet any assumed deadline
+- support their contributions in the future
+- to perform work on any specific element of the project
+
+## Communication
+
+### Direct 'vs Public Communications
+
+Owing to the open-source nature of the Haiku project, there are a number of public communications channels that are provided and are administered.  Certain contributors and particularly members of the development team do not want to be singled out as representing the project as a whole.  These individuals also do not want to take on an obligatory communicator role.  For this reason, please use public rather than private channels for communication.  Some situations where private communications may be appropriate;
+
+* where security or privacy concerns need to be raised and where the information should not be public
+* where another community member's behavior is inappropriate or offensive and you need to address the issue
+* where contributors are collaborating about specific details and both parties are happy to communicate in this way
+
+### Inappropriate Content and Behaviour
+
+Discussions relating to subjects that are clearly off-topic such as religion, politics, gender and ethnicity should not be undertaken across any of the communications channels.  Community members should also refrain from making sexually suggestive comments or using sexual inuendo toward other community members using any of the communications channels.
```

Comment:
The previous wording was, I think purposefully, more tolerant than that on off-topic subjects. With this version we would be disallowing any discussion, for example on gender and ethnicity, which would prevent discussing things like "how to improve diversity in open source software".

It is fine to have discussions going off-topic occasionally. We love to get news from Begasus' dogs on the IRC channel, for example. But sometimes it gets out of control.

So I would prefer to keep the previous wording of "refrain from delving into"?